docs/components/concepts/process-instance-migration.md

```diff
@@ -152,6 +154,7 @@ The following limitations exist that may be supported in future versions:
   - An element that becomes nested in a newly added sub-process
   - An element that was nested in a sub-process is no longer nested in that sub-process
 - Mapping instructions cannot change the element type
+- Mapping instructions cannot change the task implementation
```

Comment:
🔧 @korthout Would be good if we give and explanation of this. We can say something similar to "e.g. user task implementations (zeebe user task and job based user task)".